example/lib/movas/router/router.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:movas/router/router.dart';
+import 'package:movas_example/movas/ui/pages/home/home_page.dart';
+
+var router = Router(
+  initialRoute: RouteName.home,
+  useDemo: false,
```

Comment:
it would be good to showcase demo routes as well, for development.

you can create demo route of one particular widget